ereuse_devicehub/resources/device/models.py

```diff
@@ -484,7 +484,6 @@ class Computer(Device):
                             db.ForeignKey(User.id),
                             nullable=True)
     receiver = db.relationship(User, primaryjoin=receiver_id == User.id)
-    deliverynote_address = db.Column(CIText(), nullable=True)
```

Comment:
I don't quite understand this change. I guess it is already the aim to unlink the deliverynote within the Computer class